src/btree/bt_handle.c

```diff
@@ -401,6 +398,12 @@ __btree_conf(WT_SESSION_IMPL *session, WT_CKPT *ckpt)
     else
         F_SET(btree, WT_BTREE_NO_LOGGING);
 
+    WT_RET(__wt_config_gets(session, cfg, "tiered_object", &cval));
```

Comment:
Further down is the `readonly` configuration that should be removed/modified/whatever once you decide which of these we need.